lib/Dialect/Torch/Transforms/MaximizeValueSemantics.cpp

```diff
@@ -244,6 +288,111 @@ class AbstractlyInterpretCopyToNonValueTensorOpUsersWithinABlock
     rewriteSlice(*interpretedOps, rewriter);
     return success();
   }
+
+private:
+  static LogicalResult
```

Comment:
Removing recursion should allow changing this to `void`, removing the need for the ignored return in `(void)calculateAliasInfo` above